test_cupy_slow.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -ex
+
+cd cupy
+python setup.py build -j 4 develop install --user || python setup.py develop install --user
+
+export PYTHONWARNINGS="ignore::FutureWarning"
```

Comment:
We are now using `pytest-warnings`. So we don't need to use this environment variable.

Reply:
Removed.  We need to remove `PYTHONWARNINGS` from others too.